CMakeLists.txt

```diff
@@ -366,7 +366,11 @@ endif()
 add_definitions(-DAUTO_INITIALIZE_EASYLOGGINGPP)
 
 # Generate header for embedded translations
-add_subdirectory(translations)
+include(ExternalProject)
+ExternalProject_Add(generate_translations_header
+  SOURCE_DIR "${CMAKE_CURRENT_SOURCE_DIR}/translations"
+  BINARY_DIR "${CMAKE_CURRENT_BINARY_DIR}/translations"
+  INSTALL_COMMAND cmake -E echo "")
```

Comment:
What does this bit with echo do ?

Reply:
The build failed when ```INSTALL_COMMAND``` was not set or set to an empty string as apparently *ExternalProject* requires an *install* step even if in this case there is nothing to install.

So as a workaround I set ```INSTALL_COMMAND``` to a command doing nothing that should be available on all architectures (printing an empty message using *cmake*).